README.md

```diff
@@ -55,6 +55,9 @@ Finally, if you prefer to avoid `conda` altogether, the following commands provi
 ```shell
 pip install git+https://github.com/catalystneuro/roiextractors.git@main
 ```
+### Special Setup for macOS ARM64
+
+When using ROIExtractors with Inscopix (`isx`) on Apple Silicon (macOS arm64 architecture), note that the `isx` package is not natively supported. The installation instructions are located at the [Inscopix installation guide](https://pypi.org/project/isx/).
```

Comment:
Actually, thinking more about this. This is too specific for inscopix, probably should not be in the general readme and the installation instructions that we give when someone tries to use should be sufficient.

What do you think?